src/schema/SchemaSelectors.js

```diff
@@ -159,3 +160,30 @@ export const hasSchemaProperty = createSelector(
   [hasProperty],
   hasProperty => hasProperty
 );
+
+export const getActions = createSelector(
+  [actions],
+  actions => actions
+);
+
+export const getDetailViewActions = createSelector(
```

Comment:
Please rename function `getSingularActions`(or something like this) because now name is related to usage.